bmemcached/protocol.py

```diff
@@ -9,6 +9,7 @@
     from urllib.parse import SplitResult  # type: ignore[import-not-found]
 
 import zlib
+from ipaddress import ip_address
```

Comment:
Let me just check with someone if it is ok to drop support for python2 and I'll review your pr, thanks for that!